G2P.pm

```diff
@@ -232,7 +238,9 @@ my $a_keys = {
   "gnomADe" => 1,
   "gnomADe_r2.1.1" => 1,
   "gnomADg" => 1,
-  "gnomADg_v3.1.2" => 1
+  "gnomADg_v3.1.2" => 1,
+  "gnomADg_v4.1" => 1,
+  "gnomADe_v4.1" => 1
```

Comment:
There is a message in [L352-354](https://github.com/Ensembl/VEP_plugins/blob/cc9e022adfa0e86ed90f209666e77e59c074b79c/G2P.pm#L352-L354) with the list of supported keys. The new keys gnomADg_v4.1 and gnomADe_v4.1 should also be included there.

In the future, we should change the way keys are checked.